plugin/manager_linux.go

```diff
@@ -169,6 +169,8 @@ func (pm *Manager) disable(p *v2.Plugin, c *controller) error {
 
 	c.restart = false
 	shutdownPlugin(p, c, pm.containerdClient)
+	pluginDir := filepath.Join(pm.config.Root, p.GetID())
+	mount.RecursiveUnmount(pluginDir)
```

Comment:
Actually I think this may be overkill since unmounts happen in `StateChanged` when we get the containerd event.